client/my-sites/upgrades/checkout/credits-payment-box.jsx

```diff
@@ -46,14 +46,9 @@ var CreditsPaymentBox = React.createClass( {
 	},
 
 	render: function() {
-		var classSet = classNames( {
-			'credits-payment-box': true,
-			selected: this.props.selected === true
-		} );
-
 		return (
 			<PaymentBox
-				classSet={ classSet }
+				classSet="credits-payment-box"
```

Comment:
I've implemented the css animation using `@keyframes` instead of adding/removing a `selected` class. As far as I can see we shouldn't have any issues with this.